tests/rule_based_profiler/data_assistant/test_column_value_missing_data_assistant.py

```diff
@@ -11,6 +19,109 @@
 )
 
 
+@pytest.fixture
+def sqlite_datasource_name() -> str:
+    return "sqlite_datasource"
+
+
+@pytest.fixture
+def sqlite_database_path() -> pathlib.Path:
+    relative_path = pathlib.Path(
+        "..",
+        "..",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    )
+    return pathlib.Path(__file__).parent.joinpath(relative_path).resolve(strict=True)
+
+
+@pytest.fixture
+def sqlite_datasource(
+    empty_data_context, sqlite_database_path, sqlite_datasource_name
+) -> SqliteDatasource:
+    connection_string = f"sqlite:///{sqlite_database_path}"
+    datasource = SqliteDatasource(
+        name=sqlite_datasource_name,
+        connection_string=connection_string,  # type: ignore[arg-type]  # pydantic will coerce
+    )
+    empty_data_context.add_datasource(
+        datasource=datasource,
+    )
+    return datasource
+
+
+@pytest.mark.integration
+@pytest.mark.slow
+def test_single_batch_not_null_expectation(sqlite_datasource):
+    asset = sqlite_datasource.add_query_asset(
+        name="my_asset", query="select vendor_id from yellow_tripdata_sample_2019_01"
+    )
+    data_context = sqlite_datasource._data_context
```

Comment:
If we also pass our `context` fixture, could we avoid accessing the private `data_context` field?  Thanks.